src/main/java/controller/RacingGameController.java

```diff
@@ -0,0 +1,58 @@
+package controller;
+
+import domain.Cars;
+import domain.GameCount;
+import view.InputView;
+import view.OutputView;
+
+public class RacingGameController {
+    private GameCount count;
+    private Cars cars;
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public RacingGameController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void init() {
+        while (!readCarNames()) ;
+        while (!readTryCount()) ;
+    }
```

Comment:
```suggestion
    public void init() {
        readCarNames() ;
        readTryCount() ;
    }
```
이런 느낌이면 읽기 더 좋을 것 같습니다.